lib/code-manager.js

```diff
@@ -126,14 +144,17 @@ export function getRegexString() {
   return regexString;
 }
 
-export function getBreakpoints() {
-  const buffer = store.editor.getBuffer();
-  const breakpoints = [buffer.getFirstPosition()];
+export function getBreakpoints(editor: atom$TextEditor) {
+  const buffer = editor.getBuffer();
+  const breakpoints = [new Point(0, 0)];
```

Comment:
Oh yeah this is a good one. I went with the hard code because we would have needed another flow definition and in the end the entire 2 line source for `getFirstPosition` reads:

```coffeescript
getFirstPosition: ->
    new Point(0, 0)

```